test/runtests.jl

```diff
@@ -22,11 +22,12 @@ if CUDA.functional() && CUDA.has_cutensor()
 end
 
 push!(my_tests,
-    "base.jl",
-   "contractions.jl",
-   "compressions.jl",
-   "ising.jl",
-   "search.jl"
+#    "base.jl",
```

Comment:
All tests should be uncommented before merging

Reply:
uncommented